src/MultiOwnable.sol

```diff
@@ -3,16 +3,19 @@ pragma solidity ^0.8.4;
 
 /// @custom:storage-location erc7201:coinbase.storage.MultiOwnable
 struct MultiOwnableStorage {
-    /// @dev tracks the index of the next owner added, not useful after 255 owners added.
-    uint8 nextOwnerIndex;
-    /// @dev Allows an owner to be idenfitied by a uint8.
-    /// Seco256r1 verifier does not recover the address, but requires
-    /// the X,Y coordinates to be passed for verification.
-    /// In the context of checking whether something was signed by an owner
-    /// this means that the signature needs to include an identifier of the owner.
-    /// In an effort to economize calldata, we use a uint8 rather than passing the
-    /// X,Y coordinates.
-    mapping(uint8 => bytes) ownerAtIndex;
+    /// @dev tracks the index of the next owner added
+    uint256 nextOwnerIndex;
```

Comment:
I'm opinionated on still using owner indices.

Reply:
wdym?

Reply:
I would use something like [EnumerableSet](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/structs/EnumerableSet.sol) to maintain the owners array without empty indices.

Reply:
Why?

Reply:
Ah I see your edit, fair comment! Don't want to take on that scope right now as I think this meets our requirements with no downsides. Will keep in mind.